Libplanet/Blockchain/BlockChainStates.cs

```diff
@@ -38,16 +41,61 @@ public BlockChainStates(IStore store, IStateStore stateStore)
             BlockHash offset,
             StateCompleter<T> stateCompleter)
         {
-            HashDigest<SHA256>? stateRootHash = _store.GetStateRootHash(offset);
-            if (stateRootHash is { } h && _stateStore.ContainsStateRoot(h))
+            Stopwatch stopwatch = new Stopwatch();
+            stopwatch.Start();
```

Comment:
Seems not used anywhere?

Reply:
Yup, an old artifact from testing. 😕
Just removed it.